fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -3,7 +3,7 @@
   enable_ruby
   <record>
     hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
-    message ${log}
+    message ${(message rescue nil) || log}
     version 1.1.4
   </record>
   remove_keys log,stream
```

Comment:
Should we just preserve `log` then?  I was wanting to avoid having duplicate entries to avoid confusion, but that might be better to the alternative of dropping things...